### PR TITLE
[UX][Web][SIMS #1543] Manage Users Page

### PR DESCRIPTION
**Pull request by hellolynn-tbtb**
### Manage Users Page
- New content in header (role definitions)
- Enhanced header styles for better readability on PC (rendering BCSans Bold in addition to font-weight as fallback)
- New badge border colours for higher contrast
- Table enhancements and alignments, formatting

### Additional information
- Left BCSans Bold out for smaller font sizes
- Badge border colours are derived from the main semantic colours with transparency but in HEX
- [Figma mockups (reference only)](https://www.figma.com/file/9Om2tzLAVEoMQrjmrH4fhu/MOD-SIMS?node-id=9474%3A97285&t=61Xy9Bq4brnHXkB2-0)

### Screenshot
image

### CODE REVIEW

File: sources/packages/web/src/assets/css/base.scss [hunk 1]
```diff
@@ -330,3 +349,19 @@ html * {
   background-color: $default-background;
   border: 2px solid $common-border-color !important;
 }
+
+// List style - apply to lists used in content
+ul {
+  margin-left: 20px;
+  margin-bottom: 0;
+
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/assets/css/base.scss:357]: Can we remove the extra blank line?

File: sources/packages/web/src/plugins/vuetify.ts [hunk 2]
```diff
@@ -22,10 +22,10 @@ export default createVuetify({
           warning: "#ff7a00",
           success: "#16c92e",
           error: "#e4222e",
-          "info-bg": "#f0eef9",
-          "warning-bg": "#fff9ed",
-          "success-bg": "#eeffef",
-          "error-bg": "#fdf4f4",
+          "info-bg": "#a99fda",
+          "warning-bg": "#FFC38C",
```
Comment (ann-aot) [inline sources/packages/web/src/plugins/vuetify.ts:26]: we will also need to change here, 
image

File: sources/packages/web/src/assets/css/base.scss [hunk 3]
```diff
@@ -79,15 +80,31 @@ html * {
...
 }
 
-/** Style to align records per page with arrows in data table. **/
+// Data table styles
+// Style to align records per page with arrows in data table.
 .p-paginator .p-dropdown .p-dropdown-label {
   margin-top: 8px;
 }
 
+.p-datatable.p-paginator-bottom {
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/base.scss:89]: mm.. these are primevue data table classes. maybe we can add a todo, to remove these styles, when we move to vuetify data table

File: sources/packages/web/src/assets/css/base.scss [hunk 4]
```diff
@@ -79,15 +80,31 @@ html * {
   border-color: $badge-border-error !important;
 }
 
-/** Style to align records per page with arrows in data table. **/
+// Data table styles
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/base.scss:83]: mm, we add a full stop after each sentence in our comments. we review other comments too

**Review by andrewsignori-aot — APPROVED** [review]
Nice work, looks good 👍

File: sources/packages/web/src/assets/css/base.scss (hunk 3 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/assets/css/base.scss:89]: @ann-aot I agree. I was thinking the same. You'll see I put all the p-datatable styles stuff under a comment, which we should be able to remove easily (and not think too much about it) when we change tables.

File: sources/packages/web/src/plugins/vuetify.ts (hunk 2 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/plugins/vuetify.ts:26]: @ann-aot Thanks ann. The ones in Vuetify.ts change the badge border colours which are different from banner background colours. So I believe we should leave the $banner-background-info as is.

File: sources/packages/web/src/assets/css/base.scss (hunk 4 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/assets/css/base.scss:83]: @ann-aot Thanks Ann - can you give me an example what you mean? Are you referring to the commenting style?

File: sources/packages/web/src/assets/css/base.scss [hunk 5]
```diff
@@ -79,15 +80,31 @@ html * {
...
+// Style to align records per page with arrows in data table.
```
Comment (hellolynn-tbtb) [inline sources/packages/web/src/assets/css/base.scss:84]: @ann-aot does this need to be put back using //** **// ?

File: sources/packages/web/src/assets/css/base.scss (hunk 3 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/assets/css/base.scss:89]: @ann-aot Ah okay I see we have a todo style comment now. Thank you. Done!

File: sources/packages/web/src/plugins/vuetify.ts (hunk 2 above)
Comment (ann-aot) [reply sources/packages/web/src/plugins/vuetify.ts:26]: vuetify theme is not only for badges border, it is for all the components with property `color` and `background color` like vuetify banners, buttons,snack bar, alerts etc. these styles variable are used wherever we can't use color or background-color property and sometimes even to override the vuetify default styles

File: sources/packages/web/src/assets/css/base.scss (hunk 4 above)
Comment (ann-aot) [reply sources/packages/web/src/assets/css/base.scss]: example, 
image
image
period `(.)` at the end of a sentance

File: sources/packages/web/src/assets/css/base.scss (hunk 5 above)
Comment (ann-aot) [reply sources/packages/web/src/assets/css/base.scss]: for styles, we use both.  https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#sample-class-example

Comment (hellolynn-tbtb) [reply sources/packages/web/src/assets/css/base.scss]: Thanks @ann-aot !

**Pull request merged**